tests/gui/base.py

```diff
@@ -351,11 +361,11 @@ def die(self, terminate=False):
             if terminate:
                 self._proc.terminate()
 
-            # tests will click the Reboot or Quit button which will shutdown anaconda
-            # we need to make sure /mnt/sysimage/* are unmounted and device mapper devices
-            # are removed before starting the next test. ATM I have no idea if pyanaconda
-            # provides an API for this so wait for a while instead!
-            doDelay(30)
+            # Tests will click the Reboot or Quit button which will shutdown anaconda.
+            # We need to make sure /mnt/sysimage/* are unmounted and device mapper devices
+            # are removed before starting the next test.
+            subprocess.call(["%s/scripts/anaconda-cleanup" % os.environ.get("top_srcdir", "")],
```

Comment:
If $top_srcdir isn't set set, this will mean /scripts/anaconda-cleanup will get run.  You probably want ./scripts/anaconda-cleanup to get run instead.  So:

os.environ.get("top_srcdir", ".")